spec.proto

```diff
@@ -384,6 +384,20 @@ message ProtoSpecResult {
     repeated Error errors = 10;
 }
 
+/// A proto object representing the result of Scenario execution.
+message ProtoScenarioResult {
+    ProtoScenario protoScenario = 1;
```

Comment:
Does it makes sense to have ProtoItem here instead?

Reply:
I'm not sure if we need to add it as a ProtoItem. Since the result is for scenario explicitly, I don't see a point of adding as ProtoItem.
Later if we need to ass `Steps` then they will be ProtoItem. (There will be steps, teardowns contexts).

Reply:
We have 
```
    ProtoScenario protoScenario = 1;
    ProtoTable scenarioDataTableRow = 2;
    int64 scenarioDataTableRowIndex = 3;
    ProtoTable ScenarioDataTable = 4;
```
All of which is a part of ProtoItem. 
Also, ProtoItem has types for `ProtoScenario` and `ProtoTableDrivenScenario`. That is the reason I though that ProtoItem may be a better choice.